README.md

```diff
@@ -18,57 +20,42 @@ To hash a list of bytes, invoke the [`convert`][convert] method on the
 objects.
 
 ```dart
-import 'package:crypto/crypto.dart';
 import 'dart:convert'; // for the UTF8.encode method
+import 'package:crypto/crypto.dart';
 
 void main() {
-  var bytes = UTF8.encode("foobar"); // data being hashed
+  var bytes = UTF8.encode('foobar'); // data being hashed
```

Comment:
Changing quote types adds churn for no value.

Reply:
The rest of the package has standardized on single quotes. Going for consistency

Reply:
It's not worth the effort or churn to try to be consistent about quote types.